apps/onboarding-ms/src/main/java/it/pagopa/selfcare/onboarding/util/QueryUtils.java

```diff
@@ -69,11 +70,34 @@ public static Map<String, String> createMapForOnboardingQueryParameter(String pr
         return queryParameterMap;
     }
 
+    public static Map<String, String> createMapForOnboardingReject(String reasonForReject, String onboardingStatus) {
+        Map<String, String> queryParameterMap = new HashMap<>();
+        Optional.ofNullable(reasonForReject).ifPresent(value -> queryParameterMap.put("reasonForReject", value));
+        Optional.ofNullable(onboardingStatus).ifPresent(value -> queryParameterMap.put("status", value));
```

Comment:
you could add updatedAt update

Reply:
I didn't update updateAt because I saw that the following code is executed every time on the Orchestrator side:
`public void updateOnboardingStatus(String onboardingId, OnboardingStatus status) {
        repository
                .update("status = ?1 and updatedAt = ?2", status.name(), LocalDateTime.now())
                .where("_id", onboardingId);
    }`